M2/VERSION-README

```diff
@@ -1,22 +1,18 @@
-
 The only place the Macaulay2 version number appears is in the file ./VERSION .
 
-We have three main branches on github:
+We have two main branches on github:
 
    development -- all pull requests are merged into this branch
-   pre-master  -- development is manually merged into this branch occasionally, triggering builds
-   master      -- pre-master is manually merged into this branch after builds succeed there.  Daily builds occcur here.
+   master      -- the most recent stable release
```

Comment:
I'd like to bring up the discussion of renaming `master` into `main`, `release`, or `stable` again, so you might want to hold off on this commit.

Reply:
Actually, this whole file probably should be deleted and move to a wiki page. Is there any reason to keep this in git?

Reply:
Probably not. There are at least 4 different places where various parts of the release process are documented, so combining them into just a single wiki page seems like a good idea!

Reply:
Let's delete this then. Where are the other 3?

Reply:
This file, the wiki page, `INSTALL`, and a one of the `bugs/dan` files

Reply:
The [wiki page](https://github.com/Macaulay2/M2/wiki/Internals%3A-Release-Checklist) has been updated to contain the (updated) contents of the removed files/sections.